backend/src/lang/lang.controller.ts

```diff
@@ -1,6 +1,8 @@
 import { Controller, Post, Body, UseInterceptors, FileInterceptor, UploadedFile, Query, BadRequestException } from '@nestjs/common';
 import { DialogFlowService } from './dialog-flow.service';
 import { AudioIntentParams, TextIntentParams } from './lang.dto';
+import EmployeeService from './employee.service';
+import { EmploymentContractService } from 'database/employmentContract/employmentContract.service';
```

Comment:
Bitte vor Merge den Pfad noch auf einen relativen Pfad ändern (siehe meine Kommentare in deinem anderen PR)